packages/core/src/base-chart.ts

```diff
@@ -951,4 +954,28 @@ export class BaseChart {
 	getBBox(selector: any) {
 		return this.innerWrap.select(selector).node().getBBox();
 	}
+// ================================================================================
+	// Makes chart data components more screen reader friendly
+	// ================================================================================
+	makeAccessible (element?: any, type?: any, dataList?: any) {
+		// Make chart data components tabbable
+		element.attr("tabindex", 0);
+		// Aria label contains the x value, y value, and dataset for the data element
+		if (type === "axis") {
+			element.attr("aria-label", (d) => `Label: ${d.label}, Value: ${d.value}, belongs to ${d.datasetLabel}`);
+		} else if (type === "slice") {
+			element.attr("aria-label", (d) => `Label: ${d.data.label}, Value: ${d.value},
+			percentage is ${Tools.convertValueToPercentage(d.data.value, dataList)} percent`);
```

Comment:
Seems like the presence of `dataList` means that `type === "slice"`, so that's a step towards removing the string being passed into this function